python/lsst/ts/imsim/opdMetrology.py

```diff
@@ -107,10 +102,10 @@ def getDefaultLsstWfsGQ(self):
             Detector IDs of extra-focal sensor in raft.
         """
 
-        # Field x, y for 4 WFS
+        # Field x, y for 4 WFS in the Camera Coordinate System (CCS)
         fieldWFSx = [1.176, -1.176, -1.176, 1.176]
         fieldWFSy = [1.176, 1.176, -1.176, -1.176]
```

Comment:
Would it be possible to use FIELD_ANGLE instead of hardcoding these values? If so, we could do the same from ts_ofc

Reply:
Yeah, will update this to do that.